addons/base_vat/tests/test_validate_ruc.py

```diff
@@ -3,7 +3,7 @@
 from odoo.exceptions import ValidationError
 
 
-class TestRUCStructure(common.TransactionCase):
+class TestStructure(common.TransactionCase):
```

Comment:
The file name was intended for PE validation only. Maybe the file needs to be renamed as well?

Reply:
I just tired but the file changes too much and git can not detect the renaming so we lose the blame on `test_peru_ruc_format`.